lesson_15/inc/walker.php

```diff
@@ -83,4 +83,4 @@ function end_lvl(){ // closing ul
 	}
 */
 	
-}
```

Comment:
its really strange I didn't make this change, GitHub has done this automatically. Alex do you have any idea about this change?